idunn/api/directions.py

```diff
@@ -40,7 +40,7 @@ def get_directions(
 
         if not allowed_zone:
             raise HTTPException(
-                status_code=403,
+                status_code=406,
```

Comment:
I don't think that's the right status to use, 406 is about content negotiation, using the `Accept-*` HTTP headers sent by the client to determine if the server can produce a content matching the criteria (language, MIME type, etc.) https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/406

Reply:
The final choice seems to be 422.